### ISSUE TITLE
Tokenizer not activating after update

### ISSUE THREAD

**Issue by jpmiers** [MrPrimate/vtta-tokenizer#58]
Greetings,
After updating it seems that the module isn't installed correctly?
When clicking on the portrait the foundry filepicker launches instead of the tokenizer one. There are no errors in the console. I also noticed there isn't a tokenizer settings section in the module settings (I can't remember if there was one before).

I've tried uninstalling/reinstalling the module and the issue persists.

**Comment by MrPrimate** [comment]
Hi, what version of Foundry and what system and version are you using?Can you also try with no other nodules activated?

Thanks

**Comment by jpmiers** [comment]
Issue still occurs even with all modules deactivated except tokenizer. Foundry 0.8.9, pf2e 2.14.1.8153

**Comment by robertoshaolin** [comment]
I have the same problem here!

**Comment by jpmiers** [comment]
I solved my issue!
I had a gamesystem update that got corrupted, I resolved it by reinstalling the game system. However I think tokenizer was affected. Since I had multiple worlds with tokenizer installed I needed to uninstall on both worlds.. go into the data/modules folder and delete the vtta-tokenizer folder. I rebooted and reinstalled and now its good. Hope this helps anyone else having similar issues.
